styles/Editor.css

```diff
@@ -154,7 +154,7 @@
   border-bottom: 1px solid #000;
 }
 
-.ProseMirror tr:not(:first-of-type):not(:last-of-type) td:not(:last-child) {
+.ProseMirror tr td:not(:last-child) {
   border-right: 1px solid #000;
```

Comment:
should we add this also to `th`

Reply:
Based on the designs, no, we shouldn't.

![Screenshot 2024-07-15 at 2 32 02 PM](https://github.com/user-attachments/assets/5aecc2b8-6b05-4503-8815-7ae16db9cefb)

Reply:
Does that look ok to you, the header appears as one row

Reply:
It sounds fine to me. To be honest, I don't have a strong opinion on that.

![Screenshot 2024-07-15 at 2 53 05 PM](https://github.com/user-attachments/assets/e1de2ba4-ee2e-459d-afd0-e353f2d94681)